private_set_intersection/preload.bzl

```diff
@@ -65,35 +65,23 @@ def psi_preload():
     if "pybind11_bazel" not in native.existing_rules():
         http_archive(
             name = "pybind11_bazel",
-            strip_prefix = "pybind11_bazel-203508e14aab7309892a1c5f7dd05debda22d9a5",
-            urls = ["https://github.com/pybind/pybind11_bazel/archive/203508e14aab7309892a1c5f7dd05debda22d9a5.zip"],
-            sha256 = "75922da3a1bdb417d820398eb03d4e9bd067c4905a4246d35a44c01d62154d91",
+            strip_prefix = "pybind11_bazel-master",
+            urls = ["https://github.com/pybind/pybind11_bazel/archive/master.zip"],
```

Comment:
Can you point here to the exact commit currently on master, and add the sha256 sum? That way we make sure builds are reproducible and don't unexpectedly break if the master branch gets updated.